sound/soc/codecs/rt712-sdca-sdw.c

```diff
@@ -178,13 +194,15 @@ static int rt712_sdca_read_prop(struct sdw_slave *slave)
 	unsigned long addr;
 	struct sdw_dpn_prop *dpn;
 
+	sdw_slave_read_prop(slave);
+
```

Comment:
how is this related to the VA/VB support?

Reply:
@plbossart This is related to the VB capability for multi-lanes. 
I tested with the PR https://github.com/thesofproject/linux/pull/4733.
I think there is no effect on the VA.